editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -913,6 +913,26 @@
      :cell-value-factory (fn/partial table-cell-value-factory path)
      :cell-factory (fn/partial table-cell-factory column (dissoc edit :value))}))
 
+(defn custom-table-resize-policy []
+  (reify Callback
+    (call [_ resize-features]
+      (let [^TableView$ResizeFeatures resize-features resize-features
+            resized-column (.getColumn resize-features)
+            delta (.getDelta resize-features)
+            table (.getTable resize-features)
+            columns (.getColumns table)
+            total-width (.getWidth table)
+            last-column (last columns)]
+        (when resized-column
+          (let [new-width (max (.getMinWidth resized-column) (+ (.getPrefWidth resized-column) delta))]
+            (.setPrefWidth resized-column new-width)))
+        (when (and last-column (not= resized-column last-column))
+          (let [used-width (reduce + (map #(.getWidth %) (butlast columns)))
```

Comment:
Please remove reflection warnings. `#(.getWidth %)` definitely has one, but there might be more. The easiest way to check for reflection warnings is to reload the namespace (Tools -> REPL -> Load File in REPL). One downside of this approach is that Cursive usually reloads more than necessary and that breaks the REPL state. Also, reflection warnings are printed to the console when you start the app with `lein run`:
```
Reflection warning, editor/cljfx_form_view.clj:930:44 - reference to field getWidth can't be resolved.
Reflection warning, editor/cljfx_form_view.clj:932:45 - reference to field getMinWidth can't be resolved.
Reflection warning, editor/cljfx_form_view.clj:932:13 - call to method setPrefWidth can't be resolved (target class is unknown).
```

Reply:
nice trick, thx